PersonSearchDetails/Features/MPI/MPINewPatientInvalidName.feature

```diff
@@ -0,0 +1,9 @@
+﻿Feature: MPINewPatientInvalidName
```

Comment:
Doesnt appear to be a matching Step Definitions file for this Feature

Reply:
@samGreenwood2022 this test is on hold atm - me and Chris can't figure out a way to interact with a pop-up that occurs.